app/components/PaymentForm.jsx

```diff
@@ -1,5 +1,39 @@
+import '../tailwind.css';
```

Comment:
You shouldn't need to import the css into every file. If it's not working, check that something else isn't going wrong.